### DISCUSSION TITLE
Implement grep_string from telescope with mini.pick

### DISCUSSION THREAD

**Discussion by diego-velez** [nvim-mini/mini.nvim#1429] [Show and tell]
Hello!

Just as the title says, I've been trying to implement the `grep_string` picker from telescope in mini.pick. For those who don't know, the `grep_string` picker in telescope will grep for the current word under your cursor.

I looked to see if there was a way to set the initial query for a picker when opening one in mini.pick, but I didn't find one, so I came up with this:
```
local cword = vim.fn.expand '<cword>'
vim.defer_fn(function()
    vim.api.nvim_input(cword)
end, 25)
require('mini.pick').builtin.grep_live()
```

**Comment by echasnovski** [comment]
Thanks for sharing.

In most cases using `:Pick grep pattern='<cword>'` is enough. This might not work with special characters, but those probably won't match the during `<cword>` expansion.

**Reply by diego-velez** [reply]
Ooh, I didn't realize pattern was an option 😅, I feel silly now 😭 , but thanks for letting me know!
